.github/workflows/main.yml

```diff
@@ -1,25 +1,103 @@
-# This workflow will later be replaced with logic to "Generate Website Data"
-# The verify-gdrive.yml workflow file will be renamed to this one
-# We have to introduce this change in steps because GitHub gets confused until
-# we add the new workflow file to the master branch
 name: "Generate Website Data"
 on:
   workflow_dispatch:
+  push:
```

Comment:
I have not heard of limited hours.  There is a concept of throttling.  So if we are a highly active bunch, that may lead to problems.  Otherwise, we're ok.  I also have some checks to limit what is run.  If we want to, we can limit jobs/steps based on the files that were changed.  For now, I'm keeping it mostly simple, except for the piece that builds the docker image (which only occurs when required).

Reply:
Found this, "GitHub Actions usage is free for standard GitHub-hosted runners in public repositories". It's amazing how much compute & storage GH gives away for free 😄